proposed/cache.md

```diff
@@ -193,13 +193,14 @@ interface CacheItemInterface
      * although the method of serialization is left up to the Implementing
      * Library.
      *
-     * Implementing Libraries MAY provide a default TTL if one is not specified.
+     * Implementing Libraries MAY provide a default TTL if none is specified.
```

Comment:
I've no idea what language is more common.  Which version did PSR-7 use? Let's be consistent with that, whatever it is.